### DISCUSSION TITLE
Found a Fake site hosting this project or false positive ? ⚠️

### DISCUSSION THREAD

**Discussion by TecnikOfficial** [pear-devs/pear-desktop#2222] [Q&A]
recently i found a site possible phishing site copying name of this repo trying to look similar . my bitdefender traffic light extension flagged it so i didnt clicked on it 
<details>
  <summary>Click to view Screenshots</summary>
  <div>
    <img src="https://github.com/user-attachments/assets/f6f35899-719d-4660-a1ed-8a6f55216096" alt="Screenshot 1">
    <img src="https://github.com/user-attachments/assets/973d4fef-c64b-436a-9736-015dbf0b0c07" alt="Screenshot 2">
  </div>
</details>
later i tested that url in <a href="https://www.virustotal.com/gui/url/4d1a79e552c983cd0db54ffb2c5d513c82746136bf430d076968415b5adf52a2?nocache=1">virustotal</a> it flagged my 5 antivirus is this false positive or people are spreading malware using this project name @th-ch

**Comment by ArjixWasTaken** [comment]
Uhhhhhhhhhhhhhhhhhhhhhhhhhhhhh

that looks more of a github copycat than a youtube music copycat
so it would apply to all github projects, not just youtube music

sadly I cannot confirm anything since I cannot visit that website
